larpandoracontent/LArCheating/CheatingBeamParticleIdTool.cc

```diff
@@ -25,23 +25,23 @@ CheatingBeamParticleIdTool::CheatingBeamParticleIdTool() :
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
-void CheatingBeamParticleIdTool::SelectOutputPfos(const pandora::Algorithm *const /*pAlgorithm*/, const SliceHypotheses &nuSliceHypotheses, const SliceHypotheses &crSliceHypotheses, PfoList &selectedPfos)
+void CheatingBeamParticleIdTool::SelectOutputPfos(const pandora::Algorithm *const /*pAlgorithm*/, const SliceHypotheses &testBeamSliceHypotheses, const SliceHypotheses &crSliceHypotheses, PfoList &selectedPfos)
 {
-    if (nuSliceHypotheses.size() != crSliceHypotheses.size())
+    if (testBeamSliceHypotheses.size() != crSliceHypotheses.size())
         throw StatusCodeException(STATUS_CODE_INVALID_PARAMETER);
 
-    for (unsigned int sliceIndex = 0, nSlices = nuSliceHypotheses.size(); sliceIndex < nSlices; ++sliceIndex)
+    for (unsigned int sliceIndex = 0, nSlices = testBeamSliceHypotheses.size(); sliceIndex < nSlices; ++sliceIndex)
     {
         float beamParticleWeight(0.f), totalWeight(0.f);
-        const PfoList &neutrinoPfoList(nuSliceHypotheses.at(sliceIndex));
+        const PfoList &neutrinoPfoList(testBeamSliceHypotheses.at(sliceIndex));
```

Comment:
Did you want this container to be named `testBeamPfoList` too?

Reply:
Oh yeah good spot!  I'll correct that now.